polaris.shopify.com/src/components/Examples/Examples.module.scss

```
@@ -1,68 +1,49 @@
-.SelectContainer {
-  position: relative;
-  display: inline-block;
-  width: 100%;
-
-  select {
-    width: 100%;
-    height: 2.5rem;
-    line-height: 2.5rem;
-    padding-left: 0.75rem;
-    padding-right: 0.75rem;
-    background-color: var(--surface-subdued);
-    color: var(--text);
-    border-radius: var(--border-radius-500);
-    font-size: var(--font-size-400);
-    box-shadow: 0 0 0 1px var(--border-color);
-
-    &:focus-visible {
-      box-shadow: var(--focus-outline);
+.ExamplesList {
+  display: flex;
+  flex-wrap: wrap;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 0.5rem;
+  margin-bottom: 1rem;
+
+  button {
+    padding: 0.6rem 0.85rem;
+    text-align: left;
+    display: flex;
+    align-items: flex-start;
+    background: transparent;
+    box-shadow: var(--card-shadow);
+    border-radius: var(--border-radius-400);
+    font-size: var(--font-size-100);
+    line-height: 1.33;
+    color: var(--text-strong);
+
+    &:hover {
+      box-shadow: var(--card-shadow-hover);
     }
-  }
-}
 
-.SelectIcon {
-  pointer-events: none;
-  position: absolute;
-  top: calc(50% + 2px);
-  right: 1rem;
-  transform: translateY(-50%);
-  z-index: 1;
-
-  svg {
-    height: 1.2rem;
-    width: 1.2rem;
+    &[aria-selected="true"] {
+      background: var(--search-highlight-color);
+    }
```

Right now there's no indication visually when keyboard focus moves to the active tab that the tabs are in the page's tab order, let's add a focus ring to the active tab:

```suggestion
      &:focus {
        box-shadow: var(--focus-outline);
      }
    }  
```